cmd/manager/main.go

```diff
@@ -19,78 +19,68 @@ package main
 import (
 	"flag"
 	"net/http"
-	_ "net/http/pprof"
-	"time"
-
-	"k8s.io/klog"
-	"k8s.io/klog/klogr"
-	capa "sigs.k8s.io/cluster-api-provider-aws/pkg/apis"
-	"sigs.k8s.io/cluster-api-provider-aws/pkg/controller"
-	"sigs.k8s.io/cluster-api-provider-aws/pkg/record"
-	capi "sigs.k8s.io/cluster-api/pkg/apis"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+	"os"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/apis/infrastructure/v1alpha2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	// +kubebuilder:scaffold:imports
 )
 
+var (
+	myscheme = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
```

Comment:
Are these coming from kubebuilder?

Reply:
it's `scheme` by default but that shadows the import so I changed it to `myscheme` instead.

The setupLog is there by default in v2 main.go though.

Reply:
We should probably open an issue in kubebuilder